{
  "candidate": "CVE-2023-36053",
  "description": "In Django 3.2 before 3.2.20, 4 before 4.1.10, and 4.2 before 4.2.3, EmailValidator and URLValidator are subject to a potential ReDoS (regular expression denial of service) attack via a very large number of domain name labels of emails and URLs.",
  "priority": "medium",
  "cvss": {
    "nvd": " CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]"
  },
  "bugs": [
    "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1040225"
  ],
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-36053",
    "https://www.openwall.com/lists/oss-security/2023/07/03/1",
    "https://www.djangoproject.com/weblog/2023/jul/03/security-releases/",
    "https://github.com/django/django/commit/ad0410ec4f458aa39803e5f6b9a3736527062dcd (main)",
    "https://github.com/django/django/commit/454f2fb93437f98917283336201b4048293f7582 (3.2.20)",
    "https://docs.djangoproject.com/en/4.2/releases/security/",
    "https://groups.google.com/forum/#!forum/django-announce",
    "https://ubuntu.com/security/notices/USN-6203-1",
    "https://ubuntu.com/security/notices/USN-6203-2"
  ],
  "assigned_to": "mdeslaur, leosilva",
  "discovered_by": "Seokchan Yoon",
  "public_date": "2023-07-03 13:15:00 UTC",
  "public_date_at_usn": "2023-07-04",
  "packages": {
    "bionic": {
      "python-django": {
        "name": "python-django",
        "status": "ignored",
        "note": "end of standard support"
      }
    },
    "devel": {
      "python-django": {
        "name": "python-django",
        "status": "not-affected",
        "note": "3:3.2.20-1.1"
      }
    },
    "esm-infra/bionic": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "1:1.11.11-1ubuntu1.21+esm1"
      }
    },
    "esm-infra/xenial": {
      "python-django": {
        "name": "python-django",
        "status": "needs-triage"
      }
    },
    "focal": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "2:2.2.12-1ubuntu0.18"
      }
    },
    "jammy": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "2:3.2.12-2ubuntu1.7"
      }
    },
    "kinetic": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "3:3.2.15-1ubuntu1.4"
      }
    },
    "lunar": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "3:3.2.18-1ubuntu0.3"
      }
    },
    "mantic": {
      "python-django": {
        "name": "python-django",
        "status": "not-affected",
        "note": "3:3.2.20-1.1"
      }
    },
    "trusty": {
      "python-django": {
        "name": "python-django",
        "status": "ignored",
        "note": "end of standard support"
      }
    },
    "trusty/esm": {
      "python-django": {
        "name": "python-django",
        "status": "needs-triage"
      }
    },
    "upstream": {
      "python-django": {
        "name": "python-django",
        "status": "released",
        "note": "4.2.3,4.1.10,3.2.20"
      }
    },
    "xenial": {
      "python-django": {
        "name": "python-django",
        "status": "ignored",
        "note": "end of standard support"
      }
    }
  }
}
